### PR TITLE
[google-apps-script] Add missing definitions

### PR DESCRIPTION
**Pull request by sinsky**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - https://developers.google.com/apps-script/reference/spreadsheet/sheet?hl=en#getdrawings
  - https://developers.google.com/apps-script/reference/spreadsheet/drawing?hl=en
  - https://developers.google.com/apps-script/reference/spreadsheet/range?hl=en#getfontcolorobject
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

I apologize if there is any mistake in my first pull request.

### CODE REVIEW

File: types/google-apps-script/google-apps-script.spreadsheet.d.ts [hunk 1]
```diff
@@ -1463,6 +1480,8 @@ declare namespace GoogleAppsScript {
       setDataValidation(rule: DataValidation | null): Range;
       setDataValidations(rules: (DataValidation | null)[][]): Range;
       setFontColor(color: string | null): Range;
+      setFontColorObject(color: Color): Range;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.spreadsheet.d.ts:1483]:
```suggestion
      setFontColorObject(color: Color | null): Range;
```

> The font color to set; a null value resets the color.
https://developers.google.com/apps-script/reference/spreadsheet/range?hl=en#setFontColorObject(Color)

File: types/google-apps-script/google-apps-script.spreadsheet.d.ts [hunk 2]
```diff
@@ -1463,6 +1480,8 @@ declare namespace GoogleAppsScript {
...
+      setFontColorObjects(colors: (Color|null)[][]): Range;
```
Comment (mtgto) [inline types/google-apps-script/google-apps-script.spreadsheet.d.ts:1484]:
```suggestion
      setFontColorObjects(colors: (Color | null)[][]): Range;
```

**Review by mtgto — COMMENTED** [review]
Almost LGTM 👍

**Comment by sinsky** [comment]
@mtgto  thank you  review ❤️

**Review by mtgto — APPROVED** [review]
LGTM. Thank you for your contribution 👍

**Review by mtgto — APPROVED** [review]
Hmm… Why Typescript Bot changes the state…?
Re-Approve

**Review by mtgto — APPROVED** [review]

**Comment by sinsky** [comment]
Ready to merge

**Pull request merged**